runner/ansible/roles/post-results/tasks/main.yml

```diff
@@ -1,14 +1,14 @@
 ---
 
-- name: set_global_results {{ test_id }}
+- name: set_global_results {{ id }}
   set_fact:
     results: |
       {{ (results | default({})) | combine(
         {
           'results': {
             item: {
```

Comment:
with the rewrite I am seeing:
TASK [post-results : set_global_results 1.4.2] *******************************************************************
[WARNING]: The loop variable 'item' is already in use. You should set the `loop_var` value in the `loop_control`
option for the task to something else to avoid variable collisions and unexpected behavior.
[WARNING]: The loop variable 'item' is already in use. You should set the `loop_var` value in the `loop_control`
option for the task to something else to avoid variable collisions and unexpected behavior.

since item is also being use as a loop var in check.yml and meta.yml this is causing the warning.

Reply:
Thanks! I will have a look

Reply:
Fixed on: https://github.com/trento-project/trento/pull/255/commits/e54decdc127415b9e2a9c65542c2091395dbd1f6